packages/component/package.json

```diff
@@ -69,7 +69,7 @@
     "react-film": "3.0.0",
     "react-redux": "7.2.3",
     "react-say": "2.0.2-master.ee7cd76",
-    "react-scroll-to-bottom": "4.1.0",
+    "react-scroll-to-bottom": "4.1.1-master.300a16c",
```

Comment:
What's the plan for updating to an official release? 4.14? Could you file a bug and add it to the milestone?

Reply:
I strongly prefer that the changes to `react-scroll-to-bottom` are also reviewed. Sync?

Reply:
It's production now, 4.1.1. It's mainly this PR https://github.com/compulim/react-scroll-to-bottom/pull/85.

I added the test harness plus some "happy path" tests, thus the diff is pretty long, [`v4.1.0...v4.1.1`](https://github.com/compulim/react-scroll-to-bottom/compare/v4.1.0...v4.1.1).